wave_viewer.m

```diff
@@ -1940,6 +1946,38 @@ function expand_btw_ax_tmarkers(ax,tAx,fAx)
 end
 end
 
+function expand_btw_ax_uev(ax,tAx,fAx) 
+[~,t_spec,~,t_user_events] = get_ax_tmarker_times(ax);
+if length(t_user_events) >= 2
+    t_low = t_user_events(1) - 0.05;
+    t_hi = t_user_events(end) + 0.05;
+elseif length(t_user_events) == 1
+    t_low = t_user_events(1) - 0.2;
+    t_hi = t_user_events(1) + 0.2;
+else
+    return;
+end
+if t_low > t_hi
+    warning('First user event must precede last user event.')
+    return
+end
```

Comment:
In addition to this, we need to check if t_low is negative or t_high is greater than the length of the file.  There's some odd behavior when the user event is less than 200 ms from the beginning or end; the green/red markers go offscreen and it takes some tricks to get them back.

Reply:
We also need a check like lines 1942-1946 (old lines 1936-1940) to make sure to put the cursor between the green/red markers:
```Matlab
if t_tmarker_spec < new_t_low || t_tmarker_spec > new_t_hi
    update_tmarker(axinfo.h_tmarker_spec,(new_t_low + new_t_hi)/2);
else
    update_tmarker(axinfo.h_tmarker_spec,[]);
end
```

Reply:
This should happen already, since `expand_btw_ax_tmarkers` is called at the end of the quickzoom function. The if statement you quote is being performed. Let me know if the results with the yellow bar aren't what you expect.

Reply:
I'm getting the yellow line moving to be within the red/green for normal operations in timeAdapt events, not sure if there's some edge case that isn't working